.github/workflows/ci.yml

```diff
@@ -74,6 +74,7 @@ jobs:
     - name: Setup database
       env:
         RAILS_ENV: test
+        ACG_CONFIG: ./test.json
```

Comment:
I think you need a different file than the `test.json` here and below...one that contains the correct credentials for github's CI instead of the local test environment.